content/_index.md

```diff
@@ -14,7 +14,7 @@ It is a privacy-focused, open-source [fork][fork] of **Maps.me** app (previously
 
 > In 2023, Organic Maps [got its first million](@/news/2023-12-23/281/index.md) users. [Help us](@/donate/index.md) to scale!
 
-### Download and install Organic Maps from [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery] {#install}
+### Download and install Organic Maps from [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery], [Gentoo Linux through Guru overlay][gentoo] {#install}
```

Comment:
Not sure that dev/alpha linux version should be promoted here. It would be better to reference it at the bottom...